Source/WebCore/css/query/GenericMediaQueryParser.cpp

```diff
@@ -29,6 +29,11 @@
 #include "CSSCustomPropertyValue.h"
 #include "CSSParserImpl.h"
 #include "CSSPropertyParser.h"
+#include "CSSPropertyParserConsumer+Ident.h"
+#include "CSSPropertyParserConsumer+Integer.h"
+#include "CSSPropertyParserConsumer+Length.h"
+#include "CSSPropertyParserConsumer+Number.h"
+#include "CSSPropertyParserConsumer+Resolution.h"
 #include "CSSPropertyParserHelpers.h"
```

Comment:
Do we still need to import `CSSPropertyParserHelpers` ?

Reply:
That is still needed.